### DISCUSSION TITLE
Mobile

### DISCUSSION THREAD

**Discussion by Developer354** [community/community#127724] [Mobile] [answered]
### Select Topic Area

General

### Body

[README.txt](https://github.com/user-attachments/files/15741959/README.txt)
content/account-and-profile/index.md---
title: Getting started with writing and formatting on GitHub
redirect_from:
  - /articles/markdown-basics
  - /articles/things-you-can-do-in-a-text-area-on-github
  - /articles/getting-started-with-writing-and-formatting-on-github
  - /github/writing-on-github/getting-started-with-writing-and-formatting-on-github
intro: 'You can use simple features to format your comments and interact with others in issues, pull requests, and wikis on GitHub.'
versions:
  fpt: '*'
  ghes: '*'
  ghec: '*'
children:
  - /quickstart-for-writing-on-github
  - /about-writing-and-formatting-on-github
  - /basic-writing-and-formatting-syntax
shortTitle: Start writing on GitHub
---

**Comment by stevepopovich** [comment]
Hi @Developer354 do you have a specific question about GitHub?

**Comment by Developer354** [comment, marked as answer]
Yes, I do.
